src/platform/site-wide/side-nav/components/SideNav.js

```diff
@@ -147,6 +147,7 @@ class SideNav extends Component {
           )}
           onClick={this.toggleUlClass}
           id="sidenav-menu"
+          aria-label="In this section menu"
```

Comment:
Would it make sense to make the `ul#va-sidenav-ul-container` have `role="menu"` and add `aria-controls="va-sidenav-ul-container"`?

Reply:
@Rleahy22 I'd prefer not to add a `role="menu"` attribute. Menus require a full suite of keyboard navigation controls including arrow keys: https://www.w3.org/WAI/GL/wiki/Using_ARIA_menus

A menu might also disrupt the parent/child hierarchy of nested lists.

`aria-controls` has very limited use (JAWS is the only SR that supports it) and falls afoul if the ID it references appears and disappears. I'd opt to skip that one too.